cohorts/load.py

```diff
@@ -1262,7 +1262,11 @@ def plot_survival(self,
         )
         return results
 
-    def plot_joint(self, on, on_two=None, x_col=None, plot_type="jointplot", stat_func=pearsonr, show_stat_func=True, jointplot_kwargs={}, **kwargs):
+    def plot_joint(self, *args, **kwargs):
+        warnings.warn("plot_joint is deprecated; use plot_correlation.", Warning)
```

Comment:
Wanted this to be `DeprecationWarning`, but that doesn't show by default in notebooks and I'm not clear what the best way is to change that setting. Open to suggestions!